### ISSUE TITLE
bind dynamic value 

### ISSUE THREAD

**Issue by mamadza** [Wikiki/bulma-calendar#157]
Hi, I have problem with set value when get from server 
i set value like this in vuejs its work:
`data() {
     return {
           data: '2019-03-25'
     }
}

<input class="input selectDate" v-validate="'required'" name="datePicker" type="text" :value="date" readonly>
`
but when I try data from server with exact format the bulma don't show the value 

- [ no ] Are you running the latest version?
- [yes ] Are you reporting to the correct repository?

## Bug Report

**Environment**

- Bulma version: [5.1.4]
- OS: [Windows 10]
- Browser: [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]

**Comment by Wikiki** [comment]
Hi,

any reason you can't use the last version ?

**Comment by mamadza** [comment]
yes I change some customize calender if I get the new version I have problem , how other solve this problem in last version?
